kratos/containers/vector_map.h

```diff
@@ -535,7 +535,7 @@ class VectorMap
     {
         key_type mKey;
     public:
-        EqualKeyTo(key_type k) : mKey(k) {}
+        explicit EqualKeyTo(key_type k) : mKey(k) {}
```

Comment:
are u sure of this one? it looks a little drastic to me...

Reply:
Maybe it's overdoing it here. This didn't have any effect other than to silence the static analysis tool. If you want I can revert it.

Reply:
i met it to your judgement...it looks a bit extreme to me, but your call